### ISSUE TITLE
Lack of Screen Reader Accomodation

### ISSUE THREAD

**Issue by emma-coronado** [libevent/libevent.github.com#8]
Compatibility with screen readers is missing or can be improved for some components including that the libevent logo is missing alt text, the benchmark images could be better described with aria labels and liked to their captions, among other accessibility upgrades.

Anything to improve accessibility would be a great upgrade.

Some info about aria can be found on mdn webdocks: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA

**Comment by Kavipatel0** [comment]
I would like to work on this issue. In addition to addressing the mentioned accessibility concerns, I could also focus on replacing some div elements with semantic HTML landmarks to improve accessibility for screen readers. Additionally, the captions on images could be enhanced by encapsulating img tags within figure tags.
